lib/govuk_app_config/govuk_data_sync.rb

```diff
@@ -0,0 +1,31 @@
+require "time"
+
+class GovukDataSync
+  def initialize(govuk_data_sync_period)
+    @from, @to = false
```

Comment:
nil feels a more idiomatic value than false here. I actually think this is redundant though as I think the begin will have hoisted the ivars already into scope.

P.S. I already wrote a comment on this but it seems to have gone somewhere, apologies if you get two.